geomesa-tools/bin/common-functions.sh

```diff
@@ -17,7 +17,7 @@ function setGeoLog() {
     export GEOMESA_OPTS="-Dgeomesa.log.dir=${GEOMESA_LOG_DIR} ${GEOMESA_OPTS}"
```

Comment:
The export GEOMESA_OPTS needs to be set

Reply:
is there another change that needs to be made? this pr was already merged